scripts/_ynh_secure_remove.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+#=================================================
+# EXPERIMENTAL HELPERS
+#=================================================
+
+# Remove a file or a directory securely
+#
+# usage: ynh_secure_remove --file=path_to_remove [--regex=regex to append to $file] [--non_recursive] [--dry_run]
+# | arg: -f, --file - File or directory to remove
+# | arg: -r, --regex - Regex to append to $file to filter the files to remove
+# | arg: -n, --non_recursive - Perform a non recursive rm and a non recursive search with the regex
+# | arg: -d, --dry_run - Do not remove, only list the files to remove
+#
+# Requires YunoHost version 2.6.4 or higher.
+ynh_secure_remove () {
```

Comment:
Uuuuuh, really ?

Is this needed for the new extra options ? x_X

I guess it's nice if at some point this can be integrated into the core if this is needed/used by several apps :sweat_smile: